src/intTest/java/uk/nhs/digital/nhsconnect/lab/results/IntegrationBaseTest.java

```diff
@@ -32,10 +37,16 @@
 @Slf4j
 public abstract class IntegrationBaseTest {
 
+    public static final String DLQ_PREFIX = "DLQ.";
     protected static final int WAIT_FOR_IN_SECONDS = 10;
     protected static final int POLL_INTERVAL_MS = 100;
     protected static final int POLL_DELAY_MS = 10;
+    private static final int JMS_RECEIVE_TIMEOUT = 500;
+    @Value("${labresults.amqp.meshInboundQueueName}")
+    protected String meshInboundQueueName;
 
+    @Autowired
+    protected JmsTemplate jmsTemplate;
     @Autowired
```

Comment:
Could we add a space before this line to make it clear which annotations belong to which fields?

Reply:
Is it more readable now? 925e2df7e6146fe3407b21182cd2653193960593 Thanks!

Reply:
Looks good to me!